sky/clouds/service_catalog/__init__.py

```diff
@@ -128,6 +128,11 @@ def accelerator_in_region_or_zone(
                                acc_name, acc_count, region, zone)
 
 
+def regions(clouds: CloudFilter = None) -> 'List[cloud.Region]':
+    """Returns a list of regions."""
```

Comment:
```suggestion
    """Returns the list of regions in a Cloud's catalog."""
```
Is this correct? Cloud or clouds (forgot the details of CloudFilter)?

Does each Region object have the `zones` field filled?

Reply:
Sounds good!